src/common/legacy-parser/src/analyzer/analyzer_expr_sync.rs

```diff
@@ -13,27 +13,25 @@
 // limitations under the License.
 
 use async_trait::async_trait;
-use common_ast::udfs::UDFDefinition;
-use common_ast::udfs::UDFExprTraverser;
-use common_ast::udfs::UDFExprVisitor;
-use common_ast::udfs::UDFFetcher;
-use common_ast::udfs::UDFParser;
-use common_ast::udfs::UDFTransformer;
 use common_datavalues::prelude::*;
 use common_datavalues::type_coercion::merge_types;
 use common_exception::ErrorCode;
 use common_exception::Result;
 use common_functions::aggregates::AggregateFunctionFactory;
-use common_functions::is_builtin_function;
-use common_meta_types::UserDefinedFunction;
 use common_planners::Expression;
+use sqlparser::ast::BinaryOperator;
```

Comment:
cc @andylokandy, maybe we should also implement udf in the new plnner?

Reply:
It's already done.